# HG changeset patch
# User Julien Bect <julien.bect@supelec.fr>
# Date 1390116358 -3600
#      Sun Jan 19 08:25:58 2014 +0100
# Branch gui-release
# Node ID 6e8188effddff8773d770f21dd11f26332720004
# Parent  7ac2a8b758fcf45b670f8e9a2c9b9b14a00e6fab
file-editor-tab.cc: Fix a typo.

diff --git a/libgui/src/m-editor/file-editor-tab.cc b/libgui/src/m-editor/file-editor-tab.cc
--- a/libgui/src/m-editor/file-editor-tab.cc
+++ b/libgui/src/m-editor/file-editor-tab.cc
@@ -1,8 +1,9 @@
+
 /*
 
 Copyright (C) 2011-2013 Jacob Dawid
 
 This file is part of Octave.
 
 Octave is free software; you can redistribute it and/or modify it
 under the terms of the GNU General Public License as published by the
@@ -1231,17 +1232,17 @@ file_editor_tab::save_file_check_spaces 
 {
   QFileInfo file = QFileInfo(file_name);
 
   if (file.suffix () == "m" && file.baseName ().contains (' '))
     {
       int ans = QMessageBox::question (0, tr ("Octave Editor"),
          tr ("It is not advisable to save an Octave script\n"
               "in a file with a name containing spaces.\n\n"
-              "Do you wnat to chose another name?"),
+              "Do you want to choose another name?"),
           QMessageBox::Yes | QMessageBox::No, QMessageBox::Yes);
 
       if (ans == QMessageBox::Yes)
         return true;
     }
 
   return false;
 }
